react-with-json-logic/src/testCaseRunner/testCaseRunner.ts

```diff
@@ -0,0 +1,139 @@
+import { QuestionnaireEngine, Result } from "covquestions-js";
+import { Questionnaire, TestCase } from "covquestions-js/models/Questionnaire.generated";
+
+type TestResultSuccess = {
+  description: string;
+  success: true;
+};
+
+type TestResultError = {
+  description: string;
+  success: false;
+  errorMessage: string;
+};
+
+export type TestResult = TestResultSuccess | TestResultError;
+
+export function runTestCases(testQuestionnaire: Questionnaire): TestResult[] {
+  if (testQuestionnaire.testCases === undefined || testQuestionnaire.testCases.length === 0) {
+    return [];
+  }
+
+  return testQuestionnaire.testCases.map((testCase) => runOneTestCase(testQuestionnaire, testCase));
+}
+
+export function runOneTestCase(testQuestionnaire: Questionnaire, testCase: TestCase): TestResult {
+  const timeOfExecution = testCase.options?.fillInDate ? Date.parse(testCase.options?.fillInDate) / 1000 : undefined;
```

Comment:
I guess, we are using the division by 1000 at multiple places now. Maybe we should move that to some kind of util function which returns seconds till 1970?